rasa/engine/graph.py

```diff
@@ -361,24 +371,19 @@ def parent_node_names(self) -> List[Text]:
         """The names of the parent nodes of this node."""
         return list(self._inputs.values())
 
-    @staticmethod
```

Comment:
I think deleting this proves Kathrin's idea was great 💯 😄

Reply:
Agreed 🔥